clients/python/bcdb/auth.py

```diff
@@ -0,0 +1,59 @@
+import base64
+import time
+
+import grpc
+
+from typing import NamedTuple
+
+from mnemonic import Mnemonic
+from nacl.signing import SigningKey
+
+
+class AuthOptions(NamedTuple):
+    id: int
+    key: SigningKey
+    expires: int
+
+
+EnMnemonic = Mnemonic("english")
+
+
+def get_auth_header(options):
+    created = int(time.time())
+    expires = created + options.expires
+
+    headers = f"(created): {created}\n"
+    headers += f"(expires): {expires}\n"
+    headers += f"(key-id): {options.id}"
+
+    signed_headers = options.key.sign(headers.encode())
+    signature = base64.standard_b64encode(signed_headers.signature).decode()
+
+    auth_header = f'Signature keyId="{options.id}",algorithm="hs2019",created="{created}",expires="{expires}",headers="(created) (expires) (key-id) ", signature="{signature}"'
```

Comment:
remove extra spaces like `, signature=` between the comma and signature. and also in the `(key-id) "` before the `"`